api/v3/Grant.php

```diff
@@ -54,6 +54,7 @@ function civicrm_api3_grant_create($params) {
  *   Array of parameters determined by getfields.
  */
 function _civicrm_api3_grant_create_spec(&$params) {
+  $params['contact_id']['api.required'] = 1;
```

Comment:
I created https://issues.civicrm.org/jira/browse/CRM-19543 for the associated issue we discovered...